ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitter.scala

```diff
@@ -276,7 +276,9 @@ private[kvutils] class TransactionCommitter(
             )
         } catch {
           case err: Err.MissingInputState =>
-            logger.warn("Exception during model conformance validation.", err)
+            logger.warn(
+              s"Model conformance validation failed due to a missing input state (most likely due to invalid state on the partcipant), correlationId=${transactionEntry.commandId}"
+            )
```

Comment:
Out of scope for this PR but we should probably use contextual logging in the committers.

Reply:
Definitely. I'm thinking of making that change now.

Reply:
See https://github.com/digital-asset/daml/pull/9516.